client/client-multi/client-c/src/extensions/configuration/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2014-2016 CyberVision, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set(EXTENSION_CONFIGURATION_SOURCE_FILES
+    ${CMAKE_CURRENT_LIST_DIR}/kaa_configuration_manager.c)
+
+set(EXTENSION_CONFIGURATION_HEADER_FILES
+    ${CMAKE_CURRENT_LIST_DIR}/kaa_configuration_manager.h)
+
+add_library(extension_configuration ${EXTENSION_CONFIGURATION_SOURCE_FILES})
+target_include_directories(extension_configuration PUBLIC ${CMAKE_CURRENT_LIST_DIR})
```

Comment:
in this way *_private.h headers are exposed too. Consider using export/* directory to separate private and public headers